_posts/2020-08-26-gsoc-webgpu.md

```diff
@@ -0,0 +1,89 @@
+---
+layout:     post
+title:      GSoC wrap-up - Implementing WebGPU in Servo
+date:       2020-08-26 00:30:00
+summary:    A walk through the implementation of WebGPU in Servo
+categories: 
+---
+
+##Introduction
+Hello everyone! I am Kunal([@kunalmohan](https://github.com/kunalmohan)), an undergrad student at Indian Institute of Technology Roorkee, India. As a part of Google Summer of Code(GSoC) 2020, I worked on implementing WebGPU in Servo under the mentorship of Mr. Dzmitry Malyshau([@kvark](https://github.com/kvark)). I devoted the past 3 months working on ways to bring the API to fruition in Servo, so that Servo is able to run the existing examples and pass the Conformance Test Suite(CTS). This is going to be a brief account of how I started with the project, what challenges I faced and how I overcame them.
+
+##What is WebGPU?
+WebGPU is a future web standard, cross-platform graphics API aimed to make GPU capabilities more accessible on the web. WebGPU is designed from the ground up to efficiently map to the Vulkan, Direct3D 12, and Metal native GPU APIs. A native implementation of the API in Rust is present in the [wgpu crate](https://github.com/gfx-rs/wgpu). Servo implementation of the API uses this crate.
+
+##The Project
```

Comment:
since later you are frequently referring to the "server side", it would be nice to have a sentence somewhere here saying that Servo is a multi-process browser, and GPU is accessed from a different process than the one running the page content and scripts.